tools/cli/src/generate.rs

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use std::process::Command;
+
+const CARGO_GENERATE_GENERATE: &str = "generate";
+const MARINE_TEMPLATE_URL: &str = "https://github.com/fluencelabs/marine-template";
```

Comment:
`cargo-generate` doesn't support tags, and I think that it's Ok to stay with this variant for a while, because the main responsibility is on `cargo-generate` and won't change our template dramatically.